tests/app/template/test_rest.py

```diff
@@ -611,20 +611,33 @@ def test_create_a_template_with_foreign_service_reply_to(admin_request, sample_u
     )
 
 
-def test_get_template_reply_to(client, sample_letter_template):
+@pytest.mark.parametrize('template_default, service_default',
+                         [('template address', 'service address'),
+                          (None, 'service address'),
+                          ('template address', None),
+                          ])
```

Comment:
do we ever get a case where a service has no letter block at all? (ie: `(None, None)`)